components/automate-ui/src/app/pages/node-details/node-details.component.spec.ts

```diff
@@ -40,7 +40,7 @@ describe('NodeDetailsComponent', () => {
     describe('on initialization', () => {
       it('subscribes to the node-details event service', () => {
         spyOn(eventService.showModal$, 'subscribe')
-          .and.returnValue(new BehaviorSubject<boolean>(false));
+          .and.returnValue(new Subscription);
```

Comment:
The upgrade picked up that the return on this was actually a Subscription and needed to be updated as such.